saturn-pytorch/environment.yml

```diff
@@ -0,0 +1,32 @@
+name: saturn
+channels:
+- pytorch
+- fastai
+- defaults
+- conda-forge
+dependencies:
+- blas=*=mkl
+- bokeh
+- dask=2.30.0
+- distributed=2.30.1
+- fastai
+- ipykernel
+- ipywidgets
+- matplotlib
+- numpy
+- pandas
```

Comment:
Can we get graphviz in here? I use it an awful lot to show .visualize() for dask task graphs.